src/theme/teamcraft-theme.scss

```diff
@@ -0,0 +1,88 @@
+//Importing this file allows the use of the $teamcraft-theme variable
+//You can use this variable in every way as $light-theme or $dark-theme
+//Updated 10/13/2017 by Delaxiox
+
+//   Custom Function - Creates map of a single color hex
+@function hex-to-mat-color($hex){
```

Comment:
I'm not a big fan of this, the different palette shades are actually used in some components, overriding it for a single-color palette seems kinda strange to me (if I understand well)